packages/upload-client/src/types.ts

```diff
@@ -31,7 +31,7 @@ export interface Service {
   store: {
     add: ServiceMethod<StoreAdd, StoreAddResponse, never>
     list: ServiceMethod<StoreList, ListResponse<StoreListResult>, never>
-    remove: ServiceMethod<StoreRemove, null, never>
+    remove: ServiceMethod<StoreRemove, void, never>
```

Comment:
I think we actually can not use `void` because IPLD can not encode `undefined`, which is why it was `null`. That said I think it's best to define such returns as `{}` as it allows future extensions in backwards compatible way.

```suggestion
    remove: ServiceMethod<StoreRemove, {}, never>
```